tests/countries/test_bhutan.py

```
@@ -0,0 +1,114 @@
+        self.assertHolidayName(
+            "རྒྱལ་པོའི་འཁྲུངས་སྐར།༼ཉིན་གསུམ་པ།༽", (f"{year}-02-23" for year in range(1970, 2050))
+        )
+
+    def test_birth_anniversary_third_druk_gyalpo(self):
+        self.assertHolidayName(
+            "གསུམ་པ་དབུ་མའི་འཁྲུངས་སྐར་དུས་ཆེན།", (f"{year}-05-02" for year in range(1970, 2050))
+        )
+
+    def test_coronation_king_jigme_khesar(self):
+        self.assertHolidayName(
+            "རྒྱལ་པོ་འཇིགས་མེད་ཁེ་སར་རྣམ་རྒྱལ།ཀོ་རོ་ན་ཤི་ཡ།", (f"{year}-11-01" for year in range(1970, 2050))
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)